Models/Correction.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Fiks.Models
+{
+    public class Correction
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        [StringLength(80)]
+        public string Feedback { get; set; }
```

Comment:
Co dát jako feedback soubor?

Typicky účastník submitne PDFko, tak já si ho můžu vytisknout, ručně/na grafickém tabletu přečmárat a vrátit zpátky. (A 80 znaků je way too low)

Jako soubor to máme i teďka

Reply:
@MistrMxyzptlk  (pardon, asi jsem to předtím nemarknul jako review)

Co sem místo stringu dát file?